docs/project_setup.md

```diff
@@ -0,0 +1,50 @@
+## Step 1: Basic Project Setup
+
+A great guide to starting off the project can be found [here](http://www.agileventures.org/articles/project-setup-new-users) (use https://github.com/AgileVentures/WebsiteOne as the project URL).
+
+## Step 2: Run the setup script
+
+Get the project dependencies by running these 2 commands in a bash shell terminal
+
+    wget https://github.com/AgileVentures/setup-scripts/raw/develop/scripts/rails_setup.sh
+    WITH_PHANTOMJS=true REQUIRED_RUBY=2.2.2  source rails_setup.sh
```

Comment:
hmm, REQUIRED_RUBY in the script itself is 2.1.1 - I wonder if we need this script any more?  I'm trying a clean install on el capitan with just bundle